engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -447,7 +450,8 @@ namespace dmGameSystem
                 {
                     info->m_Version = component.m_BufferVersion;
 
-                    CopyBufferToVertexBuffer(br->m_Buffer, info->m_VertexBuffer, br->m_Stride, br->m_ElementCount, dmGraphics::BUFFER_USAGE_DYNAMIC_DRAW);
+                    dmBuffer::UpdateBounds(br->m_Buffer, dmHashString64("position"));
```

Comment:
Also handle the case where it fails to calculate a bounding box.
(should we log an error? should it still be visible?)

Reply:
Errors are logged by the UpdateMeshBounds(). Resolving this as out of date.